docs/docusaurus.config.js

```diff
@@ -52,7 +54,54 @@ module.exports = {
   projectName: config.projectSlug, // Usually your repo name.
   themeConfig: {
     prism: {
-      theme: require('prism-react-renderer/themes/github'),
+      theme: {
```

Comment:
@zepatrik this will be overwritten by docusaurus-template

Reply:
I actually thought of adding the styles and language definition to the template, that way we can use it in other docs as well later on.

Reply:
Feel free to put it in docusaurus-template for now

Reply:
*as is